### PR TITLE
chore: publish to npm

### PR DESCRIPTION
**Pull request by mariusc23**
This PR sets up publishing to npm instead of GitHub packages.

### CODE REVIEW

**Comment by pcraciunoiu** [comment]
@mariusc23 did you try publishing any packages? Does this need merged to run GHA and publish one to test?

**Comment by mariusc23** [comment]
@mariusc23 I didn't try publishing. Could run a prerelease on this branch or just merge it and run a full release on master.

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -48,7 +47,7 @@ jobs:
           yarn install --non-interactive --frozen-lockfile --cache-folder ${{
           steps.yarn-cache-dir-path.outputs.dir }}
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (pcraciunoiu) [inline .github/workflows/publish.yml:50]: Did you set this? I tried a prerelease, but it doesn't seem to work: https://github.com/uplift-ltd/nexus/actions/runs/3466437410/jobs/5790333619

I see the env var in actions, so I'm not sure what's up https://github.com/uplift-ltd/nexus/settings/secrets/actions

**Comment by pcraciunoiu** [comment]
From the latest publish, I take it this doesn't work yet? No worries if you'd rather come back to it in December. Or if you post next steps, maybe Camron/I can look into it

**Comment by mariusc23** [comment]
Before I upgraded the error was that it was trying to publish to GitHub packages on the second publish too. No idea why. I was trying to publish to both because if we ever want to publish private packages to GitHub packages, we could just configure the whole scope to install from ghp, not sure we can mix and match for the same org.

**Pull request merged**